### ISSUE TITLE
[A11y] Windows desktop app: JAWS screen reader unable to access content in the app

### ISSUE THREAD

**Issue by pauljacobson** [Automattic/wp-calypso#95753]
### Quick summary

We received the following report from a customer:

>In the past, I have successfully used your Wordpress.com Windows application v6.0.3 on my Windows 11 machine using the JAWS screen reader. When I downloaded v8.0.4, I found that screen reader accessibility is broken, with the JAWS virtual cursor no longer able to read anything on the screen, only saying "blank." Even reverting to v6.0.3 doesn't fix this. Would you please investigate and fix this issue as the app is now totally unusable to blind screen reader users.

### Steps to reproduce

I don't use a screen reader or Windows so I haven't been able to reproduce this issue.

### What you expected to happen

The JAWS virtual cursor should be able to read content on the screen.

### What actually happened

The JAWS virtual cursor on returns "blank" and is unable to read content on the screen.

### Impact

Some (< 50%)

### Available workarounds?

No and the platform is unusable

### Logs or notes

Reported in 8945825-zen.

This apparently affects version 8.0.4 of the app for Windows, as well as version 6.0.3 of the app after rolling back to this version.

Our customer reported that they are using Windows 11.

The customer is willing to test updated releases so I am sharing the link for this issue with the customer so they can share additional feedback, if they wish.

**Comment by Robertght** [comment]
📌 REPRODUCTION RESULTS
- Tested on the WordPress 8.0.4 app with the demo version of JAWS and Windows 11

📌 FINDINGS/SCREENSHOTS/VIDEO
- I'm unable to replicate this on my end and I have not heard "blank"  at all while going through the app or reading the posts.
  
📌 ACTIONS
- Requested author feedback

📌 Message to Author
@pauljacobson If the user can replicate this with an older version of the app, I believe it's possible that the issue might've been somewhere around their current setup or the JAWS software itself. Can you ask them to check with another reader app to see if that works?

**Comment by Robertght** [comment]
@pauljacobson I'm going to mark this as closed for now, but feel free to open it again if we hear back from the user with more details about this. Thanks!
